guides/release/reference/syntax-conversion-guide.md

```diff
@@ -102,11 +101,11 @@ export default Component.extend({
 <MyGreeting @params={{array "Hello" "World"}}>
 ```
 
-### When to use classic invocation syntax?
+### Are curly braces gone completely?
 
-Although Angle Bracket syntax is considered to be the best approach, classic invocation syntax is fine to keep using.
 In some cases, classic invocation is still required. 
-When you need direct support for positional arguments or if your components are nested within the file tree, you should still reach for those curly braces:
+
+For example, when you need direct support for positional arguments, or if your components are nested within the file tree, curly braces are still appropriate:
```

Comment:
"or if your components are nested within the file tree" is no longer true, we have released `<Nested::Support />`!

Reply:
I can't imagine that is accurate, since we don't have consensus on that syntax.